css/css-values/support/mixed-units-01.html

```diff
@@ -3,21 +3,14 @@
   <head>
     <style>
       body {
-        margin: 0;
-        font-size: 16px;
-      }
-      .box {
         background: rgb(0, 0, 255);
-        width: 100%;
-        height: 10px;
       }
-
       @media (width: calc(116px - 1em)) {
-        .box {
+        body {
           background: rgb(255, 165, 0);
         }
       }
     </style>
   </head>
-  <div class="box"></div>
+  <body></body>
```

Comment:
The script is still using `frame.contentWindow.document.querySelector(".box")`, so you need `<body class="box"></body>` or just change the script to `frame.contentWindow.document.body`.

Or even `frame.contentWindow.document.documentElement` and then you can remove the `<body></body>` (changing the selectors to `:root` or `html`).

Reply:
See the regression: https://github.com/web-platform-tests/wpt/pull/42517/checks?check_run_id=17837184682

Reply:
Good catch, thanks.

Resolved in [c18b4d1](https://github.com/web-platform-tests/wpt/pull/42517/commits/c18b4d1c4f8d4db2ef87844822c3f655d801a575)